charts/azuremonitor-containers/templates/omsagent-daemonset.yaml

```diff
@@ -111,6 +111,10 @@ spec:
        - name: CUSTOM_METRICS_ENDPOINT
          value: {{ .Values.omsagent.metricsEndpoint | quote }}
        {{- end }}
+       {{- if ne .Values.omsagent.resourceEndpoint "<your_resource_endpoint>" }}
```

Comment:
we have changed our agent name from "omsagent" to "ama-logs" so this needs to be updated.

Reply:
Updated.